sys-boot/gnu-efi/gnu-efi-3.0.6-r1.ebuild

```diff
@@ -29,6 +29,18 @@ RDEPEND=""
 QA_EXECSTACK="usr/*/lib*efi.a:* usr/*/crt*.o"
 RESTRICT="strip"
 
+pkg_pretend() {
+	if use custom-cflags; then
+		ewarn
+		ewarn "You have enabled building with USE=custom-cflags. Be aware that"
```

Comment:
This message is a bit... wordy, and i don't like introducing pkg_pretend just for this.

Could you just mask the USE flag in profiles/base/package.use.mask instead?